### DISCUSSION TITLE
2주차 회고

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#78] [General]
https://github.com/codeit-bootcamp-frontend/7-Sprint-Mission/discussions/293

이번주 현 시점까지 Team-Final을 돌아보며, Keep, Problem, Try를 적어주세요!

**Comment by JiinSeok** [comment]
# KEEP

**Reply by gayeong0916** [reply]
팀미팅을 통해 사소한 부분이라도 놓치지 않고 팀원들과 함께 논의하는 점이 좋았다. 이런 소통이 프로젝트의 완성도를 높이는 데 큰 도움이 되는 것 같다.

**Reply by engdawn** [reply]
다양한 방식의 도전과 시도, 열띤 소통이 좋았다고 생각합니다.

**Reply by JuhyeokC** [reply]
개발 과정 중 발생하는 문제 및 적용할 수 있는 최신 또는 더 나은 기술을 고민하는 것

**Reply by JiinSeok** [reply]
디스커션에 글을 올려 논의하는 것이 각자에게 생각할 시간을 주어 더 좋은 논의를 만든 것 같다

**Reply by clyde-yoon** [reply]
최적의 세팅, 각자 맡으신 부분들 노력해주셔서 너무 감사합니다. 우리팀만세

**Comment by JiinSeok** [comment]
# PROBLEM

**Reply by gayeong0916** [reply]
SSG나 SSR과 같은 기본 개념에 대해서 아직 이해가 부족한 부분이 많다고 생각한다.

**Reply by engdawn** [reply]
컨디션 난조가 전반적으로 있었다고 생각합니다.

**Reply by JuhyeokC** [reply]
개인 가사로 시간을 많이 사용해 프로젝트에 집중력이 떨어져 맡은 부분의 개발진도가 더디다.

**Reply by JiinSeok** [reply]
배포 도구 관리가 미비해서 발생한 에러가 있었다.

**Reply by clyde-yoon** [reply]
새로운 걸 접하면서 개발하다 보니, 필요한 기본보다 더 이상을 궁금해하다보니 계속 꼬였던 것 같았다.

**Comment by JiinSeok** [comment]
# TRY

**Reply by gayeong0916** [reply]
코드를 작성하면서 기본 개념도 함께 익히는 방향으로 접근 해야겠다.

**Reply by engdawn** [reply]
프로젝트를 하면서도 적절한 휴식과 운동이 좋을 것 같습니다.

**Reply by JuhyeokC** [reply]
3주차는 추석이 껴있어 3일이 휴일이지만 시간을 잘 활용해서 개발 진도를 많이 진행해야할 것 같다.

**Reply by JiinSeok** [reply]
이제는 갈피가 잡혀 각자의 컨디션과 생산성 관리가 가장 중요할 것 같다.

**Reply by clyde-yoon** [reply]
작은 코드 한줄을 쓰더라도 팀원들이 보기 편하게 직관적인 코드를 짜고 상황을 공유해야겠다!
